project-base/src/Shopsys/ShopBundle/Component/Flysystem/VolumeDriver.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Component\Flysystem;
+
+use Barryvdh\elFinderFlysystemDriver\Driver;
+use elFinder;
+use Shopsys\ShopBundle\Component\Flysystem\Plugin\GetUrl;
+
+class VolumeDriver extends Driver
+{
+    protected function configure()
+    {
+        @parent::configure();
+
+        // set thumbnails path
+        $path = $this->options['tmbPath'];
+
+        if ($path) {
+            if (!$this->fs->has($path)) {
+                if ($this->_mkdir($path, '')) {
+                    $this->_chmod($path, $this->options['tmbPathMode']);
+                } else {
+                    $path = '';
+                }
+            }
+
+            $stat = $this->_stat($path);
+
+            if ($this->_dirExists($path) && $stat['read']) {
```

Comment:
What does that mean in a human language? if is dir readable then is tmpPathWritable = $stat['write'].